clients/cloud/rust/src/producer.rs

```diff
@@ -0,0 +1,52 @@
+extern crate futures;
+extern crate rdkafka;
+
+use futures::future::Future;
+use rdkafka::error::KafkaError;
+use rdkafka::message::OwnedMessage;
+use rdkafka::producer::{FutureProducer, FutureRecord};
+use std::boxed::Box;
+
+mod utils;
+
+fn log_produce_result(
+    topic: String,
+    result: Result<(i32, i64), (KafkaError, OwnedMessage)>,
+) -> Result<(), ()> {
+    result
+        .and_then(|(p, o)| {
+            println!(
+                "Successfully produced record to topic {} partition [{}] @ offset {}",
+                topic, p, o
+            );
+            Ok(())
+        })
+        .map_err(|(err, _)| eprintln!("kafka error: {}", err))
+}
+
+fn main() -> Result<(), Box<std::error::Error>> {
+    let (topic, config) = utils::get_config()?;
+    let producer: FutureProducer = config.create()?;
+
+    let messages = (0..9)
+        .map(|msg| {
+            let value = msg.to_string();
+            println!("Preparing to produce record: {} {}", "alice", value);
+            producer.send(
+                FutureRecord::to(&topic)
+                    .payload(value.as_bytes())
```

Comment:
Should preferably be a conversion to utf-8 here, just to be clear that serialization is indeed a good idea.

Reply:
in rust, all Strings are utf-8 encoded: https://doc.rust-lang.org/std/string/struct.String.html